$NetBSD: patch-ab,v 1.1.1.1 2007/11/08 19:51:43 bjs Exp $

Some platforms (such as NetBSD) have UINT32_MAX without UINT32_MIN.
This makes sense to me, as what else could be the minimum 
--- pixman/pixman-private.h.orig	2007-10-24 12:30:17.000000000 -0400
+++ pixman/pixman-private.h
@@ -48,7 +48,10 @@
 #endif
 
 #ifndef UINT32_MIN
-# define UINT32_MIN             (0)
+# define UINT32_MIN             (0) /* XXX What else could this possibly be? */
+#endif
+
+#ifndef  UINT32_MAX
 # define UINT32_MAX             (4294967295U)
 #endif
 
